src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
```

Comment:
Let's do so some clearing 🙂 try to remove all unnecessary code or file;

Reply:
I deleted this file.
Thanks for your suggestion bellom! :).